### PR TITLE
Fix the Laverna import when unencrypted

### PR DESCRIPTION
**Pull request by jfsigrist**
#55 Fix the Laverna import when the backup is unencrypted.
I still have work to do to import the attached files who arent imported in this case.

### CODE REVIEW

File: src/scripts/components/importExport/Import.js [hunk 1]
```diff
@@ -424,7 +424,16 @@ export default class Import extends MnObject {
             const data      = JSON.parse(res);
 
             if (path[2] === 'notes') {
-                return this.importOldData('notes', data);
+                return this.importOldData('notes', data).then(() => {
+                    let options = { zip, data, name: file.name };
+
+                    this.readMarkdown(options).then(() => {
+                        // console.log('import: readLegacyFile(): readMarkdown: ');
+                        // console.log(data);
```
Comment (Maks-s) [inline src/scripts/components/importExport/Import.js]: Could you erase those comments or use them with log() please ?

**Comment by Maks-s** [comment]
Hi 😺
Nice ! It seems to work fine with my Laverna 0.7.51 backups. Only file imports left and it should be good to merge 😸

**Comment by daed** [comment]
I'm actually going to have some time tonight to spend on Encryptic.  I'll give this a test run and merge after if all is well.  I'll probably even cut a new release so that some of the people reporting issues can give it a shot and hopefully  we can kill a few issues off.

Thanks for the help!

File: src/scripts/components/importExport/Import.js (hunk 1 above)
Comment (jfsigrist) [reply src/scripts/components/importExport/Import.js]: @Maks-s I changed them for log()

**Comment by jfsigrist** [comment]
I've tried to import the files attached to a note.

But I can't find any of those file within the exported laverna-backup.zip, whether it's encrypted or not. I have the feeling that laverna 0.5.0 isn't backuping those files.

Which is also confirmed by the laverna exportProfile function here where the files are missing: https://github.com/Laverna/laverna/blob/master/app/scripts/modules/importExport/controller.js#L179

**Pull request merged**